<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPrabhupadaLoginWindow</class>
 <widget class="QDialog" name="QPrabhupadaLoginWindow">
  <property name="modal" stdset="0">
   <bool>false</bool>
  </property>
  <property name="windowTitle" stdset="0">
   <string>Вход в &quot;Словарь Шрилы Прабхупады&quot;</string>
  </property>
  <property name="windowModality" stdset="0">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="windowIcon" stdset="0">
   <iconset>
    <normaloff>:/Icon/resources/PrabhupadaDictionary.ico</normaloff>
   </iconset>
  </property>
  <property name="mouseTracking" stdset="0">
   <bool>false</bool>
  </property>
  <property name="geometry" stdset="0">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>273</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="LayoutPrabhupadaLogin">
   <item>
    <layout class="QHBoxLayout" name="LayoutSQL">
     <item>
      <widget class="QRadioButton" name="radioButtonSQLite">
       <property name="text" stdset="0">
        <string>SQLite</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonPostgreSQL">
       <property name="text" stdset="0">
        <string>PostgreSQL</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="LayoutConnection">
     <item row="0" column="0">
      <widget class="QLabel" name="LabelUserName">
       <property name="text" stdset="0">
        <string>Имя пользователя</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="ComboBoxUserName">
       <property name="editable" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="LabelPassword">
       <property name="text" stdset="0">
        <string>Пароль</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="LineEditPassword">
       <property name="echoMode" stdset="0">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="clearButtonEnabled" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="LabelDatabaseName">
       <property name="text" stdset="0">
        <string>Имя базы данных</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="ComboBoxDatabaseName">
       <property name="editable" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="LabelHostName">
       <property name="text" stdset="0">
        <string>Сервер</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="ComboBoxHostName">
       <property name="editable" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="LabelPort">
       <property name="text" stdset="0">
        <string>Порт</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="ComboBoxPort">
       <property name="editable" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="CheckBoxResetSettings">
       <property name="text" stdset="0">
        <string>Сбросить настройки</string>
       </property>
       <property name="whatsThis" stdset="0">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Этот флаг сбрасывает все настройки пользователя после входа в программу!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="ComboBoxSchema">
       <property name="editable" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="LabelSchema">
       <property name="text" stdset="0">
        <string>Схема</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="LabelLanguageUI">
       <property name="text" stdset="0">
        <string>Язык программы</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="ComboBoxLanguageUI"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons" stdset="0">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::FirstButton|QDialogButtonBox::Ok</set>
     </property>
     <property name="orientation" stdset="0">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioButtonSQLite</tabstop>
  <tabstop>radioButtonPostgreSQL</tabstop>
  <tabstop>ComboBoxUserName</tabstop>
  <tabstop>LineEditPassword</tabstop>
  <tabstop>ComboBoxDatabaseName</tabstop>
  <tabstop>ComboBoxHostName</tabstop>
  <tabstop>ComboBoxPort</tabstop>
  <tabstop>ComboBoxSchema</tabstop>
  <tabstop>CheckBoxResetSettings</tabstop>
 </tabstops>
 <resources>
  <include location="../PrabhupadaDictionary.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QPrabhupadaLoginWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QPrabhupadaLoginWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
